code/game/jobs/job/support_chaplain.dm

```diff
@@ -34,6 +34,8 @@
 	var/obj/item/weapon/storage/bible/B = new /obj/item/weapon/storage/bible(H)
 
 	spawn()
+		H.equip_to_slot_or_del(B, slot_l_hand)
```

Comment:
Fair enough.

This should probably be a preference in the player menu, eventually.